lib/src/core/number_to_words/number_to_words.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:persian_tools/persian_tools.dart';
+import 'package:persian_tools/src/constants/number_to_words/constants.dart';
+import 'package:persian_tools/src/internal_methods.dart';
+
+/// gets an [int] with 3 or less digits as input and converts it to persian
+String numToWord(int number) {
+  var unit = 100;
+  var result = '';
+
+  while (unit > 0) {
+    if ((number / unit).floor() * unit != 0) {
+      if (numberToWord.containsKey(number)) {
+        result += numberToWord[number]!;
+        break;
+      } else {
+        result += '${numberToWord[(number / unit).floor() * unit]} و ';
+        number %= unit;
+      }
+    }
+    unit = (unit / 10).floor();
+  }
+  return result;
+}
+
+/// checks current [input] for negative [value] and deploying converting process
+String convert(int number, bool ordinal) {
+  var result = <String>[];
+
+  final isNegative = number < 0;
+  number = isNegative ? number * -1 : number;
+
+  while (number > 0) {
+    result.add(numToWord(number % base));
+    number = (number / base).floor();
+  }
+  if (result.length > 4) return '';
+
+  for (var i = 0; i < result.length; i++) {
+    if (result[i] != '' && i != 0) result[i] += ' ${scale[i]} و ';
+  }
+  result = result.reversed.toList();
+
+  var words = result.join('');
+
+  if (words.endsWith(' و ')) words = words.substring(0, (words.length - 3));
+
+  words = trim(isNegative ? 'منفی $words' : words);
+
+  if (ordinal) words = addOrdinalSuffix(words);
+
+  return words;
+}
+
+String? numberToWordsString(String number, {bool ordinal = false}) {
+  if (number.isEmpty) return null;
+  if (number == '0') return 'صفر';
+
+  return convert(removeCommas(number).toInt(), ordinal);
+}
+
+String? numberToWordsInt(int number, {bool ordinal = false}) {
+  if (number == 0) return 'صفر';
+  return convert(number, ordinal);
+}
```

Comment:
Please change the `numberToWordsInt` to `numberToWordsInt`. It will make the package more constant. 
Checkout the [wordsToNumber](https://github.com/persian-tools/dart-persian-tools/blob/e14ac484ed9bf78e28ad5ecf1a11e6043103a9f7/lib/src/core/words_to_number/words_to_number.dart#L45) method.

Reply:
Hi, you mean change ```numberToWordsInt``` to ```numberToWords```?